geomesa-tools/bin/geomesa

```diff
@@ -36,14 +36,36 @@ if [ -z "$GEOMESA_HOME" ]; then
     export PATH=${GEOMESA_HOME}/bin:$PATH
     echo >&2 "Warning: GEOMESA_HOME is not set, using $GEOMESA_HOME"
 else
-    echo >&2 "Using GEOMESA_HOME $GEOMESA_HOME"
+    echo >&2 "Using GEOMESA_HOME as set: $GEOMESA_HOME"
+    read -p "Is this intentional? Y\n " -n 1 -r
+    if [[  $REPLY =~ ^[Nn]$ ]]; then
+        bin="$( cd -P "$( dirname "${SOURCE}" )" && cd ../ && pwd )"
+        export GEOMESA_HOME="$bin"
+        echo >&2 ""
+        echo "Now set to ${GEOMESA_HOME}"
+    fi
+    echo >&2 ""
 fi
 
 # GEOMESA paths, GEOMESA_LIB should live inside GEOMESA_HOME, but can be pointed elsewhere
 if [ -z "$GEOMESA_LIB" ]; then
     GEOMESA_LIB=${GEOMESA_HOME}/lib
+else
+    message="Warning: GEOMESA_LIB already set, probably by a prior configuration."
+    message="${message}\n Current value is ${GEOMESA_LIB}."
```

Comment:
Just caught that myself, actually. I'll twiddle it around so that the check occurs once. There's probably a need to do a bit more serious rewriting.